### ISSUE TITLE
Update API Calls for Dashboard to Get Correct Status

### ISSUE THREAD

**Issue by b-f-chan** [OHCRN/platform#653]
Update API calls for dashboard to get correct status

Currently getting it from Step 5 but need to get it from Progress endpoint (not implemented, blocked on another ticket)

**Comment by b-f-chan** [comment]
Closing, duplicate of #606
